src/commands/build2.rs

```diff
@@ -3,6 +3,7 @@ pub async fn build2(
     file: Option<&str>,
     base_url: &str,
     ignore_failed: bool,
+    no_static: bool,
```

Comment:
Not needed, instead check on file.is_some()

Reply:
Removed no-static parameter in build2()